### PR TITLE
Fix itemsBreadCrumbsData and add swagger docs for it

### PR DESCRIPTION
**Pull request by zenovich**

### CODE REVIEW

**Comment by smadbe** [comment]
Please refactor the service to achieve 100% coverage.

**Comment by zenovich** [comment]
> Please refactor the service to achieve 100% coverage.

done

File: app/api/items/get_breadcrumbs.go [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+	"net/http"
+	"strings"
+
+	"github.com/go-chi/render"
+	"github.com/jinzhu/gorm"
+
+	"github.com/France-ioi/AlgoreaBackend/app/service"
+)
+
+// swagger:operation GET /items items itemsBreadCrumbsData
```
Comment (smadbe) [inline app/api/items/get_breadcrumbs.go]: Looks weird to have `GET /items` for getting breadcrumb over an always-given set of ids. 
Don't you think we should pass the ids in the URL, so getting `GET /items/1,2,3` instead? Or even `GET /items/{item_ids:[\d,]+}/breadcrumb` as the ids cannot even be random ids (must be in hierarchy) and the data not really all item info.

Comment (zenovich) [reply app/api/items/get_breadcrumbs.go]: fixed

**Pull request merged**